### DISCUSSION TITLE
error LGHT0103 : The system cannot find the file 'C:\Users\wincompile\AppData\Local\Temp\h10zrbrk\cab_0_WixFirewallExtension\0'.

### DISCUSSION THREAD

**Discussion by MarkoKaiser** [wixtoolset/issues#6360] [Questions]
## Bugs

If this issue is a bug:

* Which version of WiX are you building with?
3.11.2.4516

* Which version of Visual Studio are you building with (if any)?
> No Visual Studio

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?
> no visual studio

* Which version of .NET are you building with?
> v4.8

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

> Windows 10 LTSC 1804

* Describe the problem and the steps to reproduce it.

> Run light.exe with -ext Firewall parameter

* Describe the behavior you expected and how it differed from the actual behavior.

> I expect that light.exe works without errors. Instead I get the following bug when calling light.exe as follows:
13:57:27	9202	Info	light.exe arguments: D:\SetupGen\Output\_DB\WXS\genmsi.wixobj -sw1050 -sw1051 -sw1076 -sice:ICE60 -fv -out D:\SetupGen\Output\_Msi\FirmwareManager.msi -ext WixNetFxExtension -ext WixFirewallExtension -b D:\SetupGen\Output\	
14:01:56	9203	Info	Windows Installer XML Toolset Linker version 3.11.2.4516
Copyright (c) .NET Foundation and contributors. All rights reserved.
C:\agent\_work\66\s\src\ext\FirewallExtension\wixlib\FirewallExtension_Platform.wxi(40) : error LGHT0103 : The system cannot find the file 'C:\Users\wincompile\AppData\Local\Temp\h10zrbrk\cab_0_WixFirewallExtension\0'.

**Comment by barnson** [comment]
WiX doesn't use issues for support. We moved this issue to the [experimental GitHub discussions area][gh].

[gh]: https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions

**Comment by robmen** [comment]
Almost invariably, this is a virus scanner preventing the WiX Toolset from processing using the %TEMP% folder. You can override WiX's temp folder using %WIX_TEMP% to set a location that isn't monitored by virus scanners.

WiX v4 makes this even easier with the `-intermediateFolder` switch (set automatically when used by MSBuild).
